packages/koa/package.json

```diff
@@ -0,0 +1,79 @@
+{
+  "name": "@hatchifyjs/koa",
+  "private": true,
+  "version": "0.0.8",
+  "description": "Fullstack CRUD Application Solution",
+  "main": "dist/exports.js",
+  "types": "dist/exports.d.ts",
+  "scripts": {
+    "prebuild": "npm run pretty:fix",
+    "prepare": "npm run build",
+    "build": "tsc",
```

Comment:
it looks like this is the command causing issues in the pipelines: https://github.com/bitovi/hatchify/actions/runs/5073496298/jobs/9112579155?pr=15

each workflow is trying to build all the packages but this package is failing on `tsc`